nextflow.config

```diff
@@ -79,7 +83,7 @@ params {
 
 // Container slug. Stable releases should specify release tag!
 // Developmental code should specify :dev
-process.container = 'nfcore/nanoseq:dev'
+process.container = 'yuukiiwa/nanoseq:0.1'
```

Comment:
```suggestion
process.container = 'nfcore/nanoseq:dev'
```

Reply:
This has to stay as `nfcore/nanoseq:dev` so we will have to update `environment.yml` with the required packages in order to build this container.

Reply:
container has been changed to nfcore/nanoseq:dev